### PR TITLE
Fix incorrect icon margin in ChangelogOverlay

### PR DESCRIPTION
**Pull request by EVAST9919**

### CODE REVIEW

**Comment by peppy** [comment]
Could you provide a screenshot showing what this fixes (before and after is super-handy).

**Comment by EVAST9919** [comment]
> Could you provide a screenshot showing what this fixes (before and after is super-handy).

Sure

**Comment by EVAST9919** [comment]
> Could you provide a screenshot showing what this fixes (before and after is super-handy).
Before:
before
After:
after

File: osu.Game/Overlays/Changelog/ChangelogBuild.cs [hunk 1]
```diff
@@ -67,21 +68,31 @@ private void load(OsuColour colours)
...
+                            Direction = FillDirection.Full,
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                        }
                     };
 
                     var entryColour = entry.Major ? colours.YellowLight : Color4.White;
 
-                    title.AddIcon(entry.Type == ChangelogEntryType.Fix ? FontAwesome.Solid.Check : FontAwesome.Solid.Plus, t =>
+                    titleContainer.Add(new SpriteIcon
```
Comment (peppy) [inline osu.Game/Overlays/Changelog/ChangelogBuild.cs]: can't this go into the above `Child =` part? (changing to `Children =`)

File: osu.Game/Overlays/Changelog/ChangelogBuild.cs [hunk 2]
```diff
@@ -67,23 +68,35 @@ private void load(OsuColour colours)
 
                 foreach (APIChangelogEntry entry in categoryEntries)
                 {
-                    LinkFlowContainer title = new LinkFlowContainer
+                    var entryColour = entry.Major ? colours.YellowLight : Color4.White;
```
Comment (iiSaLMaN) [inline osu.Game/Overlays/Changelog/ChangelogBuild.cs]: Can this be assigned directly to the icon color, it's inconsistent with how the icon is assigned: `Icon = entry.Type == ...`

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**